playbooks/roles/ip-forwarding/tasks/main.yml

```diff
@@ -1,6 +1,15 @@
 ---
-- name: Enable IPv4 traffic forwarding
+- name: "Enable IPv4 traffic forwarding"
   sysctl:
     name: net.ipv4.ip_forward
     value: 1
   when: ansible_virtualization_type != 'lxc'
+
+- name: "Add IPv4 traffic forwarding persistence service to init"
+  copy:
+    src: streisand-ipforward.sh
+    dest: /etc/init.d/streisand-ipforward
+    mode: 0755
+
+- name: "Enable the streisand-ipforward init service"
+  command: "update-rc.d streisand-ipforward defaults"
```

Comment:
FWIW, this generates a warning:
```
TASK [openconnect : Enable the streisand-openconnect init service] ******************************************************************
 [WARNING]: The service (streisand-openconnect) is actually an init script but the system is managed by systemd
```
I'm happy keeping it this way w/ the warning. It's a good reminder to write some systemd units at some point.